js/src/modal/modal.spec.js

```diff
@@ -602,6 +632,1493 @@ describe('Modal', () => {
 
       modal.show()
     })
+
+    it('should not add compensation padding when body boundary width is less than window width with no vertical scrollbar before showing and no vertical scrollbar after showing', done => {
+      const dist = document.createElement('link')
```

Comment:
I prefer if you can create a local method which inject just the CSS needed not the whole CSS please